lib/web/ui/server.go

```diff
@@ -93,7 +95,13 @@ func MakeServer(clusterName string, server types.Server, accessChecker services.
 		Addr:        server.GetAddr(),
 		Tunnel:      server.GetUseTunnel(),
 		SSHLogins:   serverLogins,
-	}, nil
+	}
+
+	if server.IsOpenSSHNode() {
+		uiServer.SubKind = "OpenSSH Server"
```

Comment:
Is there a server.GetSubKind() you can use instead?

Reply:
🤦 yes, updating